### DISCUSSION TITLE
好奇這裡的其他東西可以幹嘛

### DISCUSSION THREAD

**Discussion by 123hi123** [moesnow/March7thAssistant#312] [Q&A]
![image](https://github.com/user-attachments/assets/366cec9d-cf4a-4cd1-a551-1d2a67f86f3e)
好奇能不能刷完之後自動登出  用的是遠端桌面

**Comment by yhzcake** [comment]
可以选注销，但是不会关闭你的远程桌面软件，较好的解决办法是在检查到三月七助手关闭（可以用三月七自带的任务结束后推送消息的方式传递信息）自动停止远程桌面软件，但这需要单独写一个软件，可能需要一定编程能力

**Reply by 123hi123** [reply]
謝 回覆啦之後我有自己測試過那個注銷的功能還蠻好用的啊，不會關閉也沒關係，反正他刷完然後不要去佔用我的記憶體就很可以了
